### PR TITLE
[E2E] Morse e2e tests

### PR DESCRIPTION
**Pull request by adshmh**
PR to add E2E tests for Morse integration.

### CODE REVIEW

File: e2e/morse_relay_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// morseConfigFile is the name of the config file under "e2e" directory, that contains
+	// the config for a PATH instance that uses Shannon as the relaying protocol.
```
Comment (commoddity) [inline e2e/morse_relay_test.go:17]: NIT: should it say `uses Shannon as the relaying protocol.`?

**Review by commoddity — APPROVED** [review]
LGTM, great to have E2E for both protocols.

File: e2e/morse_relay_test.go (hunk 1 above)
Comment (adshmh) [reply e2e/morse_relay_test.go:17]: Thank you. Fixed.

**Comment by adshmh** [comment]
Closes #40

**Pull request merged**